### ISSUE TITLE
Fine-tune model

### ISSUE THREAD

**Issue by jiuliangut** [jiuliangut/earthquake-monitor#150]
### Description

Apply optimisation to hyper-parameters of the ML model to increase accuracy - (eg increasing number of folds for K-fold cross validation, adding a validation split)

---

### Outcome

Optimisations applied to the ML model to try to improve its prediction accuracy for earthquake magnitude

---

### User Story 

As a data engineer, I want to fine-tune my model to improve its performance as much as possible, so that I can be confident that it can predict earthquake magnitude as well as possible based on the training data set.
